atlas/AtlasMl/atlasml/ml/VectorEmbeddings/MainEmbeddingModel.py

```
@@ -1,9 +1,10 @@
 from openai import OpenAI
 
-client = OpenAI()
-client.api_key = "" # Replace with OpenAI API Key
-
-response = client.embeddings.create(
-    input="Task Text",
-    model="text-embedding-3-small"
-)
\ No newline at end of file
+def generate_embeddings_openai(id, description):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)